modules/rollback-operator-upgrades-console.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-operator.adoc
+:_content-type: PROCEDURE
+[id="rollback-operator-upgrades-console_{context}"]
+= Rolling back an Operator upgrade by using the web console
+
+You can roll back the Operator version by using the {ocp} web console.
+[NOTE]
+====
+This procedure can only be used for {product-title-short} release 3.74 and earlier when the `rocksdb` database is installed.
+====
+
+.Prerequisites
+
+- You have access to an {ocp} cluster web console using an account with `cluster-admin` permissions.
+
+.Procedure
+
+. Navigate to the *Operators* -> *Installed Operators* page.
+. Locate the {product-title-short} Operator and click on it.
+. On the *Operator Details* page, select *Uninstall Operator* from the *Actions* list. Following this action, the Operator stops running and no longer receives updates.
++
+[NOTE]
+====
+This action does not remove resources managed by the Operator, including custom resource definitions (CRDs) and custom resources (CRs). Dashboards and navigation items enabled by the web console and off-cluster resources that continue to run might need manual clean up. To remove these after uninstalling the Operator, you might need to manually delete the Operator CRDs.
+====
+. Determine the previous version you want to roll back to by choosing one of the following options:
+* If the current Central instance is running, you can query the {product-title-short} API to get the rollback version by running the following command:
++
+[source,terminal]
+----
+$ curl -k -s -u <user>:<pass> https://<central hostname>/v1/centralhealth/upgradestatus | jq -r .upgradeStatus.forceRollbackTo
```

Comment:
I left it in since it was awkward to send them back to the CLI instructions for just this one part, but I think it's OK - they can open a CLI window from the console (should I maybe add that instruction?)

Reply:
nah I think it's okay as-is